Makefile

```diff
@@ -48,14 +48,14 @@ _ensure-clean:
 .PHONY: release
 release: _ensure-tag _ensure-changelog _ensure-clean
 	@old_ver=`git describe --abbrev=0 --tags --match 'v[0-9]*.[0-9]*.[0-9]*'` || { echo "Failed to determine current version." >&2; exit 1; }; old_ver=$${old_ver#v}; \
-	 new_ver=`echo "$(TAG)" | sed 's/^v//'`; new_ver=$${new_ver:-patch}; \
-	 if printf "$$new_ver" | command grep -q '^[0-9]'; then \
-	   semver "$$new_ver" >/dev/null || { echo 'Invalid version number specified: $(TAG) - must be major.minor.patch' >&2; exit 2; }; \
-	   semver -r "> $$old_ver" "$$new_ver" >/dev/null || { echo 'Invalid version number specified: $(TAG) - must be HIGHER than current one.' >&2; exit 2; } \
-	 else \
-	   new_ver=`semver -i "$$new_ver" "$$old_ver"` || { echo 'Invalid version-increment specifier: $(TAG)' >&2; exit 2; } \
-	 fi; \
-	 printf "=== Bumping version **$$old_ver** to **$$new_ver** before committing and tagging:\n=== TYPE 'proceed' TO PROCEED, anything else to abort: " && read response && [ "$$response" = 'proceed' ] || { echo 'Aborted.' >&2; exit 2; };  \
-	 replace "$$old_ver" "$$new_ver" -- $(VERSIONED_FILES) && \
-	 git commit -m "v$$new_ver" $(VERSIONED_FILES) CHANGELOG.md && \
-	 git tag -a -m "v$$new_ver" "v$$new_ver"
+		new_ver=`echo "$(TAG)" | sed 's/^v//'`; new_ver=$${new_ver:-patch}; \
```

Comment:
here as well.

Reply:
Just to be clear, what for? Simply running `npm run test` will fail due to eclint error and tests wont go further.

Reply:
@ljharb As i said: https://travis-ci.org/ljharb/is-callable/jobs/551967918
Build failed cause makefile uses both, spaces and tabs

Reply:
"everyone" most certainly doesn't :-) last time someone checked github, it was about 70% spaces, 30% tabs. Separately, tabs are more flexible and more accessible than spaces.